- üëã Hi, I‚Äôm @wangbbang15
- üëÄ I‚Äôm interested in ...
- üå± I‚Äôm currently learning ...
- üíûÔ∏è I‚Äôm looking to collaborate on ...
- üì´ How to reach me ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...

<!---
wangbbang15/wangbbang15 is a ‚ú® special ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.
You can click the Preview link to take a look at your changes.
--->


#!/bin/bash

# ÌôòÍ≤Ω Î≥ÄÏàò ÏÑ§Ï†ï
source ~/.bashrc


BAM="/mnt/e/CSW/GALC"
REF_PATH="/mnt/e/CSW/reference/hg38_chromosome/chr14.fa"
DICT_PATH="/mnt/e/CSW/reference/hg38_chromosome/chr14.dict"
VCF_FILE="/mnt/e/CSW/reference/GCF_000001405.40.gz" #hg38_known_site.vcf.gz
GATK_JAR="/home/rare1/tool/gatk-4.2.0.0/gatk-package-4.2.0.0-local.jar"
PICARD_JAR="/home/rare1/gene/SNUM/picard-2.25.0/picard.jar"
THREADS=10


if [ ! -f $DICT_PATH ]; then
    echo "Creating .dict file for reference..."
    java -jar $PICARD_JAR CreateSequenceDictionary R=$REF_PATH O=$DICT_PATH
fi


if [ ! -f $VCF_FILE ]; then
    echo "Error: The VCF file ($VCF_FILE) does not exist. Please provide a valid VCF file."
    exit 1
fi


for BAM_FILE in $BAM/*_region.bam; do
    # ÌååÏùºÎ™ÖÏóêÏÑú ÏÉòÌîå Ïù¥Î¶Ñ Ï∂îÏ∂ú
    BASENAME=$(basename $BAM_FILE _region.bam)
    BAM_FILE_RG="$BAM/${BASENAME}_rg.bam"
    DEDUP_BAM="$BAM/${BASENAME}_dedup.bam"
    RECAL_TABLE="$BAM/${BASENAME}_recal_data.table"
    RECAL_BAM="$BAM/${BASENAME}_recal.bam"
    GVCF_OUTPUT="$BAM/${BASENAME}.g.vcf.gz"
    FINAL_VCF_OUTPUT="$BAM/${BASENAME}_final.vcf.gz"

    # 1. BAM ÌååÏùºÏóê Read Group Ï∂îÍ∞Ä
    echo "Adding Read Group information to $BASENAME..."
    java -jar $PICARD_JAR AddOrReplaceReadGroups \
       I=$BAM_FILE \
       O=$BAM_FILE_RG \
       RGID=$BASENAME \
       RGLB="lib1" \
       RGPL="ILLUMINA" \
       RGPU="unit1" \
       RGSM=$BASENAME

    # Ï§ëÎ≥µ Ï†úÍ±∞
    echo "Running GATK MarkDuplicates for $BASENAME..."
    java -jar $GATK_JAR MarkDuplicates \
        -I $BAM_FILE_RG \
        -O $DEDUP_BAM \
        -M $BAM/${BASENAME}_markdup_metrics.txt \
        --CREATE_INDEX true

    # GATK BaseRecalibrator 
    echo "Running GATK BaseRecalibrator for $BASENAME..."
    java -Xmx16g -Xms8g -jar $GATK_JAR BaseRecalibrator \
        -I $DEDUP_BAM \
        -R $REF_PATH \
        --known-sites $VCF_FILE \
        -O $RECAL_TABLE

    # Ïû¨ÍµêÏ†ï ÌÖåÏù¥Î∏îÏù¥ ÏÉùÏÑ±ÎêòÏóàÎäîÏßÄ ÌôïÏù∏
    if [ ! -f $RECAL_TABLE ]; then
        echo "Error: Failed to create recalibration table for $BASENAME. Skipping..."
        continue
    fi

    # GATK ApplyBQSR 
    echo "Applying BQSR for $BASENAME..."
    java -Xmx16g -Xms8g -jar $GATK_JAR ApplyBQSR \
        -I $DEDUP_BAM \
        -R $REF_PATH \
        --bqsr-recal-file $RECAL_TABLE \
        -O $RECAL_BAM

    # GATK HaplotypeCaller 
    echo "Running HaplotypeCaller for $BASENAME..."
    java -Xmx16g -Xms8g -jar $GATK_JAR HaplotypeCaller \
        -R $REF_PATH \
        -I $RECAL_BAM \
        -O $GVCF_OUTPUT \
        -ERC GVCF

    # GATK GenotypeGVCFs 
    echo "Running GenotypeGVCFs for $BASENAME..."
    java -Xmx16g -Xms8g -jar $GATK_JAR GenotypeGVCFs \
        -R $REF_PATH \
        -V $GVCF_OUTPUT \
        -O $FINAL_VCF_OUTPUT

    echo "$BASENAME GATK analysis complete. Final VCF saved to $FINAL_VCF_OUTPUT"
done

echo "All GATK analyses are complete."


Ïù¥Í±∞Î•º ÌïòÎ©¥

(base) rare1@DESKTOP-Q8E0MN3:/mnt/e/CSW/GALC/GATK$ ./GATK_test.sh
Adding Read Group information to G2110067F_GALC...
INFO    2025-02-07 09:02:17     AddOrReplaceReadGroups

********** NOTE: Picard's command line syntax is changing.
**********
********** For more information, please see:
********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)
**********
********** The command line looks like this in the new syntax:
**********
**********    AddOrReplaceReadGroups -I /mnt/e/CSW/GALC/G2110067F_GALC_region.bam -O /mnt/e/CSW/GALC/G2110067F_GALC_rg.bam -RGID G2110067F_GALC -RGLB lib1 -RGPL ILLUMINA -RGPU unit1 -RGSM G2110067F_GALC
**********


09:02:17.916 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/home/rare1/gene/SNUM/picard-2.25.0/picard.jar!/com/intel/gkl/native/libgkl_compression.so
[Fri Feb 07 09:02:17 GMT 2025] AddOrReplaceReadGroups INPUT=/mnt/e/CSW/GALC/G2110067F_GALC_region.bam OUTPUT=/mnt/e/CSW/GALC/G2110067F_GALC_rg.bam RGID=G2110067F_GALC RGLB=lib1 RGPL=ILLUMINA RGPU=unit1 RGSM=G2110067F_GALC    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false
[Fri Feb 07 09:02:17 GMT 2025] Executing as rare1@DESKTOP-Q8E0MN3 on Linux 4.4.0-17134-Microsoft amd64; OpenJDK 64-Bit Server VM 22.0.1-internal-adhoc.conda.src; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.25.0
INFO    2025-02-07 09:02:18     AddOrReplaceReadGroups  Created read-group ID=G2110067F_GALC PL=ILLUMINA LB=lib1 SM=G2110067F_GALC

[Fri Feb 07 09:02:18 GMT 2025] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.01 minutes.
Runtime.totalMemory()=536870912
Running GATK MarkDuplicates for G2110067F_GALC...
09:02:19.858 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/home/rare1/tool/gatk-4.2.0.0/gatk-package-4.2.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so
[Fri Feb 07 09:02:19 GMT 2025] MarkDuplicates --INPUT /mnt/e/CSW/GALC/G2110067F_GALC_rg.bam --OUTPUT /mnt/e/CSW/GALC/G2110067F_GALC_dedup.bam --METRICS_FILE /mnt/e/CSW/GALC/G2110067F_GALC_markdup_metrics.txt --CREATE_INDEX true --MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP 50000 --MAX_FILE_HANDLES_FOR_READ_ENDS_MAP 8000 --SORTING_COLLECTION_SIZE_RATIO 0.25 --TAG_DUPLICATE_SET_MEMBERS false --REMOVE_SEQUENCING_DUPLICATES false --TAGGING_POLICY DontTag --CLEAR_DT true --DUPLEX_UMI false --ADD_PG_TAG_TO_READS true --REMOVE_DUPLICATES false --ASSUME_SORTED false --DUPLICATE_SCORING_STRATEGY SUM_OF_BASE_QUALITIES --PROGRAM_RECORD_ID MarkDuplicates --PROGRAM_GROUP_NAME MarkDuplicates --READ_NAME_REGEX <optimized capture of last three ':' separated fields as numeric values> --OPTICAL_DUPLICATE_PIXEL_DISTANCE 100 --MAX_OPTICAL_DUPLICATE_SET_SIZE 300000 --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_MD5_FILE false --GA4GH_CLIENT_SECRETS client_secrets.json --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false
Feb 07, 2025 9:02:19 AM shaded.cloud_nio.com.google.auth.oauth2.ComputeEngineCredentials runningOnComputeEngine
INFO: Failed to detect whether we are running on Google Compute Engine.
[Fri Feb 07 09:02:20 GMT 2025] Executing as rare1@DESKTOP-Q8E0MN3 on Linux 4.4.0-17134-Microsoft amd64; OpenJDK 64-Bit Server VM 22.0.1-internal-adhoc.conda.src; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.2.0.0
INFO    2025-02-07 09:02:20     MarkDuplicates  Start of doWork freeMemory: 83769024; totalMemory: 113246208; maxMemory: 8522825728
INFO    2025-02-07 09:02:20     MarkDuplicates  Reading input file and constructing read end information.
INFO    2025-02-07 09:02:20     MarkDuplicates  Will retain up to 30879803 data points before spilling to disk.
INFO    2025-02-07 09:02:20     MarkDuplicates  Read 14336 records. 186 pairs never matched.
INFO    2025-02-07 09:02:20     MarkDuplicates  After buildSortedReadEndLists freeMemory: 219431720; totalMemory: 503316480; maxMemory: 8522825728
INFO    2025-02-07 09:02:20     MarkDuplicates  Will retain up to 266338304 duplicate indices before spilling to disk.
INFO    2025-02-07 09:02:21     MarkDuplicates  Traversing read pair information and detecting duplicates.
INFO    2025-02-07 09:02:21     MarkDuplicates  Traversing fragment information and detecting duplicates.
INFO    2025-02-07 09:02:21     MarkDuplicates  Sorting list of duplicate records.
INFO    2025-02-07 09:02:21     MarkDuplicates  After generateDuplicateIndexes freeMemory: 1902871264; totalMemory: 4068474880; maxMemory: 8522825728
INFO    2025-02-07 09:02:21     MarkDuplicates  Marking 0 records as duplicates.
INFO    2025-02-07 09:02:21     MarkDuplicates  Found 0 optical duplicate clusters.
INFO    2025-02-07 09:02:21     MarkDuplicates  Reads are assumed to be ordered by: coordinate
INFO    2025-02-07 09:02:22     MarkDuplicates  Writing complete. Closing input iterator.
INFO    2025-02-07 09:02:22     MarkDuplicates  Duplicate Index cleanup.
INFO    2025-02-07 09:02:22     MarkDuplicates  Getting Memory Stats.
INFO    2025-02-07 09:02:22     MarkDuplicates  Before output close freeMemory: 257113160; totalMemory: 293601280; maxMemory: 8522825728
INFO    2025-02-07 09:02:22     MarkDuplicates  Closed outputs. Getting more Memory Stats.
INFO    2025-02-07 09:02:22     MarkDuplicates  After output close freeMemory: 260458048; totalMemory: 293601280; maxMemory: 8522825728
[Fri Feb 07 09:02:22 GMT 2025] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 0.04 minutes.
Runtime.totalMemory()=293601280
Tool returned:
0
Running GATK BaseRecalibrator for G2110067F_GALC...
09:02:24.058 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/home/rare1/tool/gatk-4.2.0.0/gatk-package-4.2.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so
Feb 07, 2025 9:02:24 AM shaded.cloud_nio.com.google.auth.oauth2.ComputeEngineCredentials runningOnComputeEngine
INFO: Failed to detect whether we are running on Google Compute Engine.
09:02:24.221 INFO  BaseRecalibrator - ------------------------------------------------------------
09:02:24.222 INFO  BaseRecalibrator - The Genome Analysis Toolkit (GATK) v4.2.0.0
09:02:24.222 INFO  BaseRecalibrator - For support and documentation go to https://software.broadinstitute.org/gatk/
09:02:24.222 INFO  BaseRecalibrator - Executing as rare1@DESKTOP-Q8E0MN3 on Linux v4.4.0-17134-Microsoft amd64
09:02:24.223 INFO  BaseRecalibrator - Java runtime: OpenJDK 64-Bit Server VM v22.0.1-internal-adhoc.conda.src
09:02:24.224 INFO  BaseRecalibrator - Start Date/Time: February 7, 2025, 9:02:24‚ÄØAM GMT
09:02:24.224 INFO  BaseRecalibrator - ------------------------------------------------------------
09:02:24.225 INFO  BaseRecalibrator - ------------------------------------------------------------
09:02:24.226 INFO  BaseRecalibrator - HTSJDK Version: 2.24.0
09:02:24.226 INFO  BaseRecalibrator - Picard Version: 2.25.0
09:02:24.229 INFO  BaseRecalibrator - Built for Spark Version: 2.4.5
09:02:24.229 INFO  BaseRecalibrator - HTSJDK Defaults.COMPRESSION_LEVEL : 2
09:02:24.230 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false
09:02:24.230 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true
09:02:24.231 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false
09:02:24.232 INFO  BaseRecalibrator - Deflater: IntelDeflater
09:02:24.232 INFO  BaseRecalibrator - Inflater: IntelInflater
09:02:24.234 INFO  BaseRecalibrator - GCS max retries/reopens: 20
09:02:24.235 INFO  BaseRecalibrator - Requester pays: disabled
09:02:24.235 INFO  BaseRecalibrator - Initializing engine
09:02:24.526 INFO  FeatureManager - Using codec VCFCodec to read file file:///mnt/e/CSW/reference/GCF_000001405.40.gz
09:02:24.601 WARN  IndexUtils - Feature file "file:///mnt/e/CSW/reference/GCF_000001405.40.gz" appears to contain no sequence dictionary. Attempting to retrieve a sequence dictionary from the associated index file
09:02:24.720 INFO  BaseRecalibrator - Shutting down engine
[February 7, 2025, 9:02:24‚ÄØAM GMT] org.broadinstitute.hellbender.tools.walkers.bqsr.BaseRecalibrator done. Elapsed time: 0.01 minutes.
Runtime.totalMemory()=8589934592
***********************************************************************

A USER ERROR has occurred: Input files reference and features have incompatible contigs: No overlapping contigs found.
  reference contigs = [chr14]
  features contigs = [NC_000001.11, NC_000002.12, NC_000003.12, NC_000004.12, NC_000005.10, NC_000006.12, NC_000007.14, NC_000008.11, NC_000009.12, NC_000010.11, NC_000011.10, NC_000012.12, NC_000013.11, NC_000014.9, NC_000015.10, NC_000016.10, NC_000017.11, NC_000018.10, NC_000019.10, NC_000020.11, NC_000021.9, NC_000022.11, NC_000023.11, NC_000024.10, NC_012920.1, NT_113793.3, NT_113796.3, NT_113888.1, NT_113889.1, NT_113891.3, NT_113901.1, NT_113930.2, NT_113948.1, NT_113949.2, NT_167208.1, NT_167209.1, NT_167211.2, NT_167213.1, NT_167214.1, NT_167215.1, NT_167218.1, NT_167219.1, NT_167220.1, NT_167244.2, NT_167245.2, NT_167246.2, NT_167247.2, NT_167248.2, NT_167249.2, NT_167250.2, NT_167251.2, NT_187361.1, NT_187362.1, NT_187363.1, NT_187364.1, NT_187365.1, NT_187366.1, NT_187367.1, NT_187368.1, NT_187369.1, NT_187370.1, NT_187371.1, NT_187372.1, NT_187373.1, NT_187374.1, NT_187375.1, NT_187377.1, NT_187378.1, NT_187379.1, NT_187380.1, NT_187381.1, NT_187382.1, NT_187383.1, NT_187384.1, NT_187385.1, NT_187386.1, NT_187387.1, NT_187388.1, NT_187390.1, NT_187391.1, NT_187392.1, NT_187393.1, NT_187394.1, NT_187395.1, NT_187396.1, NT_187397.1, NT_187398.1, NT_187399.1, NT_187400.1, NT_187401.1, NT_187402.1, NT_187403.1, NT_187404.1, NT_187405.1, NT_187406.1, NT_187407.1, NT_187408.1, NT_187409.1, NT_187410.1, NT_187411.1, NT_187412.1, NT_187413.1, NT_187414.1, NT_187415.1, NT_187416.1, NT_187417.1, NT_187418.1, NT_187419.1, NT_187420.1, NT_187421.1, NT_187422.1, NT_187423.1, NT_187424.1, NT_187425.1, NT_187426.1, NT_187427.1, NT_187428.1, NT_187429.1, NT_187430.1, NT_187431.1, NT_187432.1, NT_187433.1, NT_187434.1, NT_187435.1, NT_187436.1, NT_187437.1, NT_187438.1, NT_187439.1, NT_187440.1, NT_187441.1, NT_187442.1, NT_187443.1, NT_187444.1, NT_187446.1, NT_187447.1, NT_187448.1, NT_187449.1, NT_187450.1, NT_187451.1, NT_187452.1, NT_187453.1, NT_187454.1, NT_187455.1, NT_187456.1, NT_187457.1, NT_187458.1, NT_187459.1, NT_187460.1, NT_187461.1, NT_187462.1, NT_187464.1, NT_187465.1, NT_187466.1, NT_187467.1, NT_187468.1, NT_187469.1, NT_187470.1, NT_187471.1, NT_187473.1, NT_187474.1, NT_187475.1, NT_187476.1, NT_187477.1, NT_187478.1, NT_187479.1, NT_187480.1, NT_187481.1, NT_187482.1, NT_187483.1, NT_187484.1, NT_187485.1, NT_187486.1, NT_187487.1, NT_187488.1, NT_187490.1, NT_187491.1, NT_187493.1, NT_187494.1, NT_187495.1, NT_187496.1, NT_187497.1, NT_187498.1, NT_187499.1, NT_187500.1, NT_187501.1, NT_187502.1, NT_187503.1, NT_187504.1, NT_187505.1, NT_187506.1, NT_187508.1, NT_187509.1, NT_187510.1, NT_187511.1, NT_187512.1, NT_187513.1, NT_187514.1, NT_187515.1, NT_187516.1, NT_187517.1, NT_187518.1, NT_187519.1, NT_187520.1, NT_187521.1, NT_187522.1, NT_187523.1, NT_187524.1, NT_187525.1, NT_187526.1, NT_187527.1, NT_187528.1, NT_187529.1, NT_187530.1, NT_187531.1, NT_187532.1, NT_187533.1, NT_187534.1, NT_187535.1, NT_187536.1, NT_187537.1, NT_187538.1, NT_187539.1, NT_187540.1, NT_187541.1, NT_187542.1, NT_187543.1, NT_187544.1, NT_187545.1, NT_187546.1, NT_187547.1, NT_187548.1, NT_187549.1, NT_187550.1, NT_187551.1, NT_187552.1, NT_187553.1, NT_187554.1, NT_187555.1, NT_187556.1, NT_187557.1, NT_187558.1, NT_187559.1, NT_187560.1, NT_187561.1, NT_187562.1, NT_187563.1, NT_187564.1, NT_187565.1, NT_187566.1, NT_187567.1, NT_187568.1, NT_187569.1, NT_187570.1, NT_187571.1, NT_187572.1, NT_187573.1, NT_187574.1, NT_187575.1, NT_187576.1, NT_187577.1, NT_187578.1, NT_187579.1, NT_187581.1, NT_187582.1, NT_187583.1, NT_187584.1, NT_187585.1, NT_187586.1, NT_187587.1, NT_187588.1, NT_187589.1, NT_187590.1, NT_187591.1, NT_187592.1, NT_187593.1, NT_187594.1, NT_187595.1, NT_187596.1, NT_187597.1, NT_187598.1, NT_187599.1, NT_187600.1, NT_187601.1, NT_187602.1, NT_187603.1, NT_187604.1, NT_187605.1, NT_187606.1, NT_187607.1, NT_187608.1, NT_187609.1, NT_187610.1, NT_187611.1, NT_187612.1, NT_187613.1, NT_187614.1, NT_187615.1, NT_187616.1, NT_187617.1, NT_187618.1, NT_187619.1, NT_187620.1, NT_187621.1, NT_187622.1, NT_187623.1, NT_187624.1, NT_187625.1, NT_187626.1, NT_187627.1, NT_187628.1, NT_187629.1, NT_187630.1, NT_187631.1, NT_187632.1, NT_187633.1, NT_187634.1, NT_187635.1, NT_187636.1, NT_187637.1, NT_187638.1, NT_187639.1, NT_187640.1, NT_187641.1, NT_187642.1, NT_187643.1, NT_187644.1, NT_187645.1, NT_187646.1, NT_187647.1, NT_187648.1, NT_187649.1, NT_187650.1, NT_187651.1, NT_187652.1, NT_187653.1, NT_187654.1, NT_187655.1, NT_187656.1, NT_187657.1, NT_187658.1, NT_187659.1, NT_187660.1, NT_187661.1, NT_187662.1, NT_187663.1, NT_187664.1, NT_187665.1, NT_187666.1, NT_187667.1, NT_187668.1, NT_187669.1, NT_187670.1, NT_187671.1, NT_187672.1, NT_187673.1, NT_187674.1, NT_187675.1, NT_187676.1, NT_187677.1, NT_187678.1, NT_187679.1, NT_187680.1, NT_187681.1, NT_187682.1, NT_187683.1, NT_187684.1, NT_187685.1, NT_187686.1, NT_187687.1, NT_187688.1, NT_187689.1, NT_187690.1, NT_187691.1, NT_187692.1, NT_187693.1, NW_003315905.1, NW_003315906.1, NW_003315907.2, NW_003315908.1, NW_003315909.1, NW_003315913.1, NW_003315914.1, NW_003315915.1, NW_003315917.2, NW_003315918.1, NW_003315919.1, NW_003315920.1, NW_003315921.1, NW_003315922.2, NW_003315928.1, NW_003315929.1, NW_003315930.1, NW_003315931.1, NW_003315934.1, NW_003315935.1, NW_003315936.1, NW_003315938.1, NW_003315939.2, NW_003315940.1, NW_003315941.1, NW_003315942.2, NW_003315943.1, NW_003315944.2, NW_003315945.1, NW_003315946.1, NW_003315952.3, NW_003315953.2, NW_003315954.1, NW_003315955.1, NW_003315956.1, NW_003315957.1, NW_003315958.1, NW_003315959.1, NW_003315960.1, NW_003315961.1, NW_003315962.1, NW_003315963.1, NW_003315964.2, NW_003315965.1, NW_003315966.2, NW_003315967.2, NW_003315968.2, NW_003315969.2, NW_003315970.2, NW_003315971.2, NW_003315972.2, NW_003571033.2, NW_003571036.1, NW_003571049.1, NW_003571050.1, NW_003571054.1, NW_003571055.2, NW_003571056.2, NW_003571057.2, NW_003571058.2, NW_003571059.2, NW_003571060.1, NW_003571061.2, NW_003871060.2, NW_003871073.1, NW_003871074.1, NW_003871091.1, NW_003871092.1, NW_003871093.1, NW_004166862.2, NW_004504305.1, NW_009646194.1, NW_009646195.1, NW_009646196.1, NW_009646197.1, NW_009646198.1, NW_009646199.1, NW_009646200.1, NW_009646201.1, NW_009646202.1, NW_009646203.1, NW_009646204.1, NW_009646205.1, NW_009646206.1, NW_009646207.1, NW_009646208.1, NW_009646209.1, NW_011332687.1, NW_011332688.1, NW_011332689.1, NW_011332690.1, NW_011332691.1, NW_011332692.1, NW_011332693.1, NW_011332694.1, NW_011332695.1, NW_011332696.1, NW_011332697.1, NW_011332698.1, NW_011332699.1, NW_011332700.1, NW_011332701.1, NW_012132914.1, NW_012132915.1, NW_012132916.1, NW_012132917.1, NW_012132918.1, NW_012132919.1, NW_012132920.1, NW_012132921.1, NW_013171799.1, NW_013171800.1, NW_013171801.1, NW_013171802.1, NW_013171803.1, NW_013171804.1, NW_013171805.1, NW_013171806.1, NW_013171807.1, NW_013171808.1, NW_013171809.1, NW_013171810.1, NW_013171811.1, NW_013171812.1, NW_013171813.1, NW_013171814.1, NW_014040925.1, NW_014040926.1, NW_014040927.1, NW_014040928.1, NW_014040929.1, NW_014040930.1, NW_014040931.1, NW_015148966.2, NW_015148967.1, NW_015148968.1, NW_015148969.2, NW_015495298.1, NW_015495299.1, NW_015495300.1, NW_015495301.1, NW_016107297.1, NW_016107298.1, NW_016107299.1, NW_016107300.1, NW_016107301.1, NW_016107302.1, NW_016107303.1, NW_016107304.1, NW_016107305.1, NW_016107306.1, NW_016107307.1, NW_016107308.1, NW_016107309.1, NW_016107310.1, NW_016107311.1, NW_016107312.1, NW_016107313.1, NW_016107314.1, NW_017363813.1, NW_017363814.1, NW_017363815.1, NW_017363816.1, NW_017363817.1, NW_017363818.1, NW_017363819.1, NW_017363820.1, NW_017852928.1, NW_017852929.1, NW_017852930.1, NW_017852931.1, NW_017852932.1, NW_017852933.1, NW_018654706.1, NW_018654707.1, NW_018654708.1, NW_018654709.1, NW_018654710.1, NW_018654711.1, NW_018654712.1, NW_018654713.1, NW_018654714.1, NW_018654715.1, NW_018654716.1, NW_018654717.1, NW_018654718.1, NW_018654719.1, NW_018654720.1, NW_018654721.1, NW_018654722.1, NW_018654723.1, NW_018654724.1, NW_018654725.1, NW_018654726.1, NW_019805487.1, NW_019805488.1, NW_019805489.1, NW_019805490.1, NW_019805491.1, NW_019805492.1, NW_019805493.1, NW_019805494.1, NW_019805495.1, NW_019805496.1, NW_019805497.1, NW_019805498.1, NW_019805499.1, NW_019805500.1, NW_019805501.1, NW_019805502.1, NW_019805503.1, NW_021159987.1, NW_021159988.1, NW_021159989.1, NW_021159990.1, NW_021159991.1, NW_021159992.1, NW_021159993.1, NW_021159994.1, NW_021159995.1, NW_021159996.1, NW_021159997.1, NW_021159998.1, NW_021159999.1, NW_021160000.1, NW_021160001.1, NW_021160002.1, NW_021160003.1, NW_021160004.1, NW_021160005.1, NW_021160006.1, NW_021160007.1, NW_021160008.1, NW_021160009.1, NW_021160010.1, NW_021160011.1, NW_021160012.1, NW_021160014.1, NW_021160015.1, NW_021160016.1, NW_021160017.1, NW_021160019.1, NW_021160020.1, NW_021160021.1, NW_021160022.1, NW_021160023.1, NW_021160024.1, NW_021160025.1, NW_021160026.1, NW_021160027.1, NW_021160028.1, NW_021160029.1, NW_021160030.1, NW_021160031.1, NW_025791753.1, NW_025791754.1, NW_025791755.1, NW_025791756.1, NW_025791757.1, NW_025791758.1, NW_025791759.1, NW_025791760.1, NW_025791761.1, NW_025791762.1, NW_025791763.1, NW_025791764.1, NW_025791765.1, NW_025791766.1, NW_025791767.1, NW_025791768.1, NW_025791769.1, NW_025791770.1, NW_025791771.1, NW_025791772.1, NW_025791773.1, NW_025791774.1, NW_025791775.1, NW_025791776.1, NW_025791777.1, NW_025791778.1, NW_025791779.1, NW_025791780.1, NW_025791781.1, NW_025791782.1, NW_025791783.1, NW_025791784.1, NW_025791785.1, NW_025791786.1, NW_025791787.1, NW_025791788.1, NW_025791789.1, NW_025791790.1, NW_025791791.1, NW_025791792.1, NW_025791793.1, NW_025791794.1, NW_025791795.1, NW_025791796.1, NW_025791797.1, NW_025791798.1, NW_025791799.1, NW_025791800.1, NW_025791801.1, NW_025791802.1, NW_025791803.1, NW_025791804.1, NW_025791805.1, NW_025791806.1, NW_025791807.1, NW_025791808.1, NW_025791809.1, NW_025791810.1, NW_025791811.1, NW_025791812.1, NW_025791813.1, NW_025791814.1, NW_025791815.1, NW_025791816.1, NW_025791817.1, NW_025791818.1, NW_025791819.1, NW_025791820.1, NW_025791821.1]

***********************************************************************
Set the system property GATK_STACKTRACE_ON_USER_EXCEPTION (--java-options '-DGATK_STACKTRACE_ON_USER_EXCEPTION=true') to print the stack trace.
Error: Failed to create recalibration table for G2110067F_GALC. Skipping...
Adding Read Group information to G2110067M_GALC...
INFO    2025-02-07 09:02:25     AddOrReplaceReadGroups

********** NOTE: Picard's command line syntax is changing.
**********
********** For more information, please see:
********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)
**********
********** The command line looks like this in the new syntax:
**********
**********    AddOrReplaceReadGroups -I /mnt/e/CSW/GALC/G2110067M_GALC_region.bam -O /mnt/e/CSW/GALC/G2110067M_GALC_rg.bam -RGID G2110067M_GALC -RGLB lib1 -RGPL ILLUMINA -RGPU unit1 -RGSM G2110067M_GALC
**********


09:02:25.562 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/home/rare1/gene/SNUM/picard-2.25.0/picard.jar!/com/intel/gkl/native/libgkl_compression.so
[Fri Feb 07 09:02:25 GMT 2025] AddOrReplaceReadGroups INPUT=/mnt/e/CSW/GALC/G2110067M_GALC_region.bam OUTPUT=/mnt/e/CSW/GALC/G2110067M_GALC_rg.bam RGID=G2110067M_GALC RGLB=lib1 RGPL=ILLUMINA RGPU=unit1 RGSM=G2110067M_GALC    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false
[Fri Feb 07 09:02:25 GMT 2025] Executing as rare1@DESKTOP-Q8E0MN3 on Linux 4.4.0-17134-Microsoft amd64; OpenJDK 64-Bit Server VM 22.0.1-internal-adhoc.conda.src; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.25.0
INFO    2025-02-07 09:02:25     AddOrReplaceReadGroups  Created read-group ID=G2110067M_GALC PL=ILLUMINA LB=lib1 SM=G2110067M_GALC

[Fri Feb 07 09:02:25 GMT 2025] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.01 minutes.
Runtime.totalMemory()=536870912
Running GATK MarkDuplicates for G2110067M_GALC...
09:02:27.501 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/home/rare1/tool/gatk-4.2.0.0/gatk-package-4.2.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so
[Fri Feb 07 09:02:27 GMT 2025] MarkDuplicates --INPUT /mnt/e/CSW/GALC/G2110067M_GALC_rg.bam --OUTPUT /mnt/e/CSW/GALC/G2110067M_GALC_dedup.bam --METRICS_FILE /mnt/e/CSW/GALC/G2110067M_GALC_markdup_metrics.txt --CREATE_INDEX true --MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP 50000 --MAX_FILE_HANDLES_FOR_READ_ENDS_MAP 8000 --SORTING_COLLECTION_SIZE_RATIO 0.25 --TAG_DUPLICATE_SET_MEMBERS false --REMOVE_SEQUENCING_DUPLICATES false --TAGGING_POLICY DontTag --CLEAR_DT true --DUPLEX_UMI false --ADD_PG_TAG_TO_READS true --REMOVE_DUPLICATES false --ASSUME_SORTED false --DUPLICATE_SCORING_STRATEGY SUM_OF_BASE_QUALITIES --PROGRAM_RECORD_ID MarkDuplicates --PROGRAM_GROUP_NAME MarkDuplicates --READ_NAME_REGEX <optimized capture of last three ':' separated fields as numeric values> --OPTICAL_DUPLICATE_PIXEL_DISTANCE 100 --MAX_OPTICAL_DUPLICATE_SET_SIZE 300000 --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_MD5_FILE false --GA4GH_CLIENT_SECRETS client_secrets.json --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false
Feb 07, 2025 9:02:27 AM shaded.cloud_nio.com.google.auth.oauth2.ComputeEngineCredentials runningOnComputeEngine
INFO: Failed to detect whether we are running on Google Compute Engine.
[Fri Feb 07 09:02:27 GMT 2025] Executing as rare1@DESKTOP-Q8E0MN3 on Linux 4.4.0-17134-Microsoft amd64; OpenJDK 64-Bit Server VM 22.0.1-internal-adhoc.conda.src; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.2.0.0
INFO    2025-02-07 09:02:27     MarkDuplicates  Start of doWork freeMemory: 83833544; totalMemory: 113246208; maxMemory: 8522825728
INFO    2025-02-07 09:02:27     MarkDuplicates  Reading input file and constructing read end information.
INFO    2025-02-07 09:02:27     MarkDuplicates  Will retain up to 30879803 data points before spilling to disk.
INFO    2025-02-07 09:02:28     MarkDuplicates  Read 16735 records. 376 pairs never matched.
INFO    2025-02-07 09:02:28     MarkDuplicates  After buildSortedReadEndLists freeMemory: 236021304; totalMemory: 520093696; maxMemory: 8522825728
INFO    2025-02-07 09:02:28     MarkDuplicates  Will retain up to 266338304 duplicate indices before spilling to disk.
INFO    2025-02-07 09:02:29     MarkDuplicates  Traversing read pair information and detecting duplicates.
INFO    2025-02-07 09:02:29     MarkDuplicates  Traversing fragment information and detecting duplicates.
INFO    2025-02-07 09:02:29     MarkDuplicates  Sorting list of duplicate records.
INFO    2025-02-07 09:02:29     MarkDuplicates  After generateDuplicateIndexes freeMemory: 1919712944; totalMemory: 4085252096; maxMemory: 8522825728
INFO    2025-02-07 09:02:29     MarkDuplicates  Marking 0 records as duplicates.
INFO    2025-02-07 09:02:29     MarkDuplicates  Found 0 optical duplicate clusters.
INFO    2025-02-07 09:02:29     MarkDuplicates  Reads are assumed to be ordered by: coordinate
INFO    2025-02-07 09:02:29     MarkDuplicates  Writing complete. Closing input iterator.
INFO    2025-02-07 09:02:29     MarkDuplicates  Duplicate Index cleanup.
INFO    2025-02-07 09:02:29     MarkDuplicates  Getting Memory Stats.
INFO    2025-02-07 09:02:29     MarkDuplicates  Before output close freeMemory: 258325456; totalMemory: 293601280; maxMemory: 8522825728
INFO    2025-02-07 09:02:29     MarkDuplicates  Closed outputs. Getting more Memory Stats.
INFO    2025-02-07 09:02:29     MarkDuplicates  After output close freeMemory: 260521864; totalMemory: 293601280; maxMemory: 8522825728
[Fri Feb 07 09:02:29 GMT 2025] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 0.04 minutes.
Runtime.totalMemory()=293601280
Tool returned:
0
Running GATK BaseRecalibrator for G2110067M_GALC...
09:02:31.629 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/home/rare1/tool/gatk-4.2.0.0/gatk-package-4.2.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so
Feb 07, 2025 9:02:31 AM shaded.cloud_nio.com.google.auth.oauth2.ComputeEngineCredentials runningOnComputeEngine
INFO: Failed to detect whether we are running on Google Compute Engine.
09:02:31.810 INFO  BaseRecalibrator - ------------------------------------------------------------
09:02:31.811 INFO  BaseRecalibrator - The Genome Analysis Toolkit (GATK) v4.2.0.0
09:02:31.811 INFO  BaseRecalibrator - For support and documentation go to https://software.broadinstitute.org/gatk/
09:02:31.812 INFO  BaseRecalibrator - Executing as rare1@DESKTOP-Q8E0MN3 on Linux v4.4.0-17134-Microsoft amd64
09:02:31.813 INFO  BaseRecalibrator - Java runtime: OpenJDK 64-Bit Server VM v22.0.1-internal-adhoc.conda.src
09:02:31.814 INFO  BaseRecalibrator - Start Date/Time: February 7, 2025, 9:02:31‚ÄØAM GMT
09:02:31.814 INFO  BaseRecalibrator - ------------------------------------------------------------
09:02:31.815 INFO  BaseRecalibrator - ------------------------------------------------------------
09:02:31.818 INFO  BaseRecalibrator - HTSJDK Version: 2.24.0
09:02:31.818 INFO  BaseRecalibrator - Picard Version: 2.25.0
09:02:31.819 INFO  BaseRecalibrator - Built for Spark Version: 2.4.5
09:02:31.819 INFO  BaseRecalibrator - HTSJDK Defaults.COMPRESSION_LEVEL : 2
09:02:31.820 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false
09:02:31.820 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true
09:02:31.821 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false
09:02:31.822 INFO  BaseRecalibrator - Deflater: IntelDeflater
09:02:31.822 INFO  BaseRecalibrator - Inflater: IntelInflater
09:02:31.823 INFO  BaseRecalibrator - GCS max retries/reopens: 20
09:02:31.823 INFO  BaseRecalibrator - Requester pays: disabled
09:02:31.824 INFO  BaseRecalibrator - Initializing engine
09:02:31.968 INFO  FeatureManager - Using codec VCFCodec to read file file:///mnt/e/CSW/reference/GCF_000001405.40.gz
09:02:32.048 WARN  IndexUtils - Feature file "file:///mnt/e/CSW/reference/GCF_000001405.40.gz" appears to contain no sequence dictionary. Attempting to retrieve a sequence dictionary from the associated index file
09:02:32.177 INFO  BaseRecalibrator - Shutting down engine
[February 7, 2025, 9:02:32‚ÄØAM GMT] org.broadinstitute.hellbender.tools.walkers.bqsr.BaseRecalibrator done. Elapsed time: 0.01 minutes.
Runtime.totalMemory()=8589934592
***********************************************************************

A USER ERROR has occurred: Input files reference and features have incompatible contigs: No overlapping contigs found.
  reference contigs = [chr14]
  features contigs = [NC_000001.11, NC_000002.12, NC_000003.12, NC_000004.12, NC_000005.10, NC_000006.12, NC_000007.14, NC_000008.11, NC_000009.12, NC_000010.11, NC_000011.10, NC_000012.12, NC_000013.11, NC_000014.9, NC_000015.10, NC_000016.10, NC_000017.11, NC_000018.10, NC_000019.10, NC_000020.11, NC_000021.9, NC_000022.11, NC_000023.11, NC_000024.10, NC_012920.1, NT_113793.3, NT_113796.3, NT_113888.1, NT_113889.1, NT_113891.3, NT_113901.1, NT_113930.2, NT_113948.1, NT_113949.2, NT_167208.1, NT_167209.1, NT_167211.2, NT_167213.1, NT_167214.1, NT_167215.1, NT_167218.1, NT_167219.1, NT_167220.1, NT_167244.2, NT_167245.2, NT_167246.2, NT_167247.2, NT_167248.2, NT_167249.2, NT_167250.2, NT_167251.2, NT_187361.1, NT_187362.1, NT_187363.1, NT_187364.1, NT_187365.1, NT_187366.1, NT_187367.1, NT_187368.1, NT_187369.1, NT_187370.1, NT_187371.1, NT_187372.1, NT_187373.1, NT_187374.1, NT_187375.1, NT_187377.1, NT_187378.1, NT_187379.1, NT_187380.1, NT_187381.1, NT_187382.1, NT_187383.1, NT_187384.1, NT_187385.1, NT_187386.1, NT_187387.1, NT_187388.1, NT_187390.1, NT_187391.1, NT_187392.1, NT_187393.1, NT_187394.1, NT_187395.1, NT_187396.1, NT_187397.1, NT_187398.1, NT_187399.1, NT_187400.1, NT_187401.1, NT_187402.1, NT_187403.1, NT_187404.1, NT_187405.1, NT_187406.1, NT_187407.1, NT_187408.1, NT_187409.1, NT_187410.1, NT_187411.1, NT_187412.1, NT_187413.1, NT_187414.1, NT_187415.1, NT_187416.1, NT_187417.1, NT_187418.1, NT_187419.1, NT_187420.1, NT_187421.1, NT_187422.1, NT_187423.1, NT_187424.1, NT_187425.1, NT_187426.1, NT_187427.1, NT_187428.1, NT_187429.1, NT_187430.1, NT_187431.1, NT_187432.1, NT_187433.1, NT_187434.1, NT_187435.1, NT_187436.1, NT_187437.1, NT_187438.1, NT_187439.1, NT_187440.1, NT_187441.1, NT_187442.1, NT_187443.1, NT_187444.1, NT_187446.1, NT_187447.1, NT_187448.1, NT_187449.1, NT_187450.1, NT_187451.1, NT_187452.1, NT_187453.1, NT_187454.1, NT_187455.1, NT_187456.1, NT_187457.1, NT_187458.1, NT_187459.1, NT_187460.1, NT_187461.1, NT_187462.1, NT_187464.1, NT_187465.1, NT_187466.1, NT_187467.1, NT_187468.1, NT_187469.1, NT_187470.1, NT_187471.1, NT_187473.1, NT_187474.1, NT_187475.1, NT_187476.1, NT_187477.1, NT_187478.1, NT_187479.1, NT_187480.1, NT_187481.1, NT_187482.1, NT_187483.1, NT_187484.1, NT_187485.1, NT_187486.1, NT_187487.1, NT_187488.1, NT_187490.1, NT_187491.1, NT_187493.1, NT_187494.1, NT_187495.1, NT_187496.1, NT_187497.1, NT_187498.1, NT_187499.1, NT_187500.1, NT_187501.1, NT_187502.1, NT_187503.1, NT_187504.1, NT_187505.1, NT_187506.1, NT_187508.1, NT_187509.1, NT_187510.1, NT_187511.1, NT_187512.1, NT_187513.1, NT_187514.1, NT_187515.1, NT_187516.1, NT_187517.1, NT_187518.1, NT_187519.1, NT_187520.1, NT_187521.1, NT_187522.1, NT_187523.1, NT_187524.1, NT_187525.1, NT_187526.1, NT_187527.1, NT_187528.1, NT_187529.1, NT_187530.1, NT_187531.1, NT_187532.1, NT_187533.1, NT_187534.1, NT_187535.1, NT_187536.1, NT_187537.1, NT_187538.1, NT_187539.1, NT_187540.1, NT_187541.1, NT_187542.1, NT_187543.1, NT_187544.1, NT_187545.1, NT_187546.1, NT_187547.1, NT_187548.1, NT_187549.1, NT_187550.1, NT_187551.1, NT_187552.1, NT_187553.1, NT_187554.1, NT_187555.1, NT_187556.1, NT_187557.1, NT_187558.1, NT_187559.1, NT_187560.1, NT_187561.1, NT_187562.1, NT_187563.1, NT_187564.1, NT_187565.1, NT_187566.1, NT_187567.1, NT_187568.1, NT_187569.1, NT_187570.1, NT_187571.1, NT_187572.1, NT_187573.1, NT_187574.1, NT_187575.1, NT_187576.1, NT_187577.1, NT_187578.1, NT_187579.1, NT_187581.1, NT_187582.1, NT_187583.1, NT_187584.1, NT_187585.1, NT_187586.1, NT_187587.1, NT_187588.1, NT_187589.1, NT_187590.1, NT_187591.1, NT_187592.1, NT_187593.1, NT_187594.1, NT_187595.1, NT_187596.1, NT_187597.1, NT_187598.1, NT_187599.1, NT_187600.1, NT_187601.1, NT_187602.1, NT_187603.1, NT_187604.1, NT_187605.1, NT_187606.1, NT_187607.1, NT_187608.1, NT_187609.1, NT_187610.1, NT_187611.1, NT_187612.1, NT_187613.1, NT_187614.1, NT_187615.1, NT_187616.1, NT_187617.1, NT_187618.1, NT_187619.1, NT_187620.1, NT_187621.1, NT_187622.1, NT_187623.1, NT_187624.1, NT_187625.1, NT_187626.1, NT_187627.1, NT_187628.1, NT_187629.1, NT_187630.1, NT_187631.1, NT_187632.1, NT_187633.1, NT_187634.1, NT_187635.1, NT_187636.1, NT_187637.1, NT_187638.1, NT_187639.1, NT_187640.1, NT_187641.1, NT_187642.1, NT_187643.1, NT_187644.1, NT_187645.1, NT_187646.1, NT_187647.1, NT_187648.1, NT_187649.1, NT_187650.1, NT_187651.1, NT_187652.1, NT_187653.1, NT_187654.1, NT_187655.1, NT_187656.1, NT_187657.1, NT_187658.1, NT_187659.1, NT_187660.1, NT_187661.1, NT_187662.1, NT_187663.1, NT_187664.1, NT_187665.1, NT_187666.1, NT_187667.1, NT_187668.1, NT_187669.1, NT_187670.1, NT_187671.1, NT_187672.1, NT_187673.1, NT_187674.1, NT_187675.1, NT_187676.1, NT_187677.1, NT_187678.1, NT_187679.1, NT_187680.1, NT_187681.1, NT_187682.1, NT_187683.1, NT_187684.1, NT_187685.1, NT_187686.1, NT_187687.1, NT_187688.1, NT_187689.1, NT_187690.1, NT_187691.1, NT_187692.1, NT_187693.1, NW_003315905.1, NW_003315906.1, NW_003315907.2, NW_003315908.1, NW_003315909.1, NW_003315913.1, NW_003315914.1, NW_003315915.1, NW_003315917.2, NW_003315918.1, NW_003315919.1, NW_003315920.1, NW_003315921.1, NW_003315922.2, NW_003315928.1, NW_003315929.1, NW_003315930.1, NW_003315931.1, NW_003315934.1, NW_003315935.1, NW_003315936.1, NW_003315938.1, NW_003315939.2, NW_003315940.1, NW_003315941.1, NW_003315942.2, NW_003315943.1, NW_003315944.2, NW_003315945.1, NW_003315946.1, NW_003315952.3, NW_003315953.2, NW_003315954.1, NW_003315955.1, NW_003315956.1, NW_003315957.1, NW_003315958.1, NW_003315959.1, NW_003315960.1, NW_003315961.1, NW_003315962.1, NW_003315963.1, NW_003315964.2, NW_003315965.1, NW_003315966.2, NW_003315967.2, NW_003315968.2, NW_003315969.2, NW_003315970.2, NW_003315971.2, NW_003315972.2, NW_003571033.2, NW_003571036.1, NW_003571049.1, NW_003571050.1, NW_003571054.1, NW_003571055.2, NW_003571056.2, NW_003571057.2, NW_003571058.2, NW_003571059.2, NW_003571060.1, NW_003571061.2, NW_003871060.2, NW_003871073.1, NW_003871074.1, NW_003871091.1, NW_003871092.1, NW_003871093.1, NW_004166862.2, NW_004504305.1, NW_009646194.1, NW_009646195.1, NW_009646196.1, NW_009646197.1, NW_009646198.1, NW_009646199.1, NW_009646200.1, NW_009646201.1, NW_009646202.1, NW_009646203.1, NW_009646204.1, NW_009646205.1, NW_009646206.1, NW_009646207.1, NW_009646208.1, NW_009646209.1, NW_011332687.1, NW_011332688.1, NW_011332689.1, NW_011332690.1, NW_011332691.1, NW_011332692.1, NW_011332693.1, NW_011332694.1, NW_011332695.1, NW_011332696.1, NW_011332697.1, NW_011332698.1, NW_011332699.1, NW_011332700.1, NW_011332701.1, NW_012132914.1, NW_012132915.1, NW_012132916.1, NW_012132917.1, NW_012132918.1, NW_012132919.1, NW_012132920.1, NW_012132921.1, NW_013171799.1, NW_013171800.1, NW_013171801.1, NW_013171802.1, NW_013171803.1, NW_013171804.1, NW_013171805.1, NW_013171806.1, NW_013171807.1, NW_013171808.1, NW_013171809.1, NW_013171810.1, NW_013171811.1, NW_013171812.1, NW_013171813.1, NW_013171814.1, NW_014040925.1, NW_014040926.1, NW_014040927.1, NW_014040928.1, NW_014040929.1, NW_014040930.1, NW_014040931.1, NW_015148966.2, NW_015148967.1, NW_015148968.1, NW_015148969.2, NW_015495298.1, NW_015495299.1, NW_015495300.1, NW_015495301.1, NW_016107297.1, NW_016107298.1, NW_016107299.1, NW_016107300.1, NW_016107301.1, NW_016107302.1, NW_016107303.1, NW_016107304.1, NW_016107305.1, NW_016107306.1, NW_016107307.1, NW_016107308.1, NW_016107309.1, NW_016107310.1, NW_016107311.1, NW_016107312.1, NW_016107313.1, NW_016107314.1, NW_017363813.1, NW_017363814.1, NW_017363815.1, NW_017363816.1, NW_017363817.1, NW_017363818.1, NW_017363819.1, NW_017363820.1, NW_017852928.1, NW_017852929.1, NW_017852930.1, NW_017852931.1, NW_017852932.1, NW_017852933.1, NW_018654706.1, NW_018654707.1, NW_018654708.1, NW_018654709.1, NW_018654710.1, NW_018654711.1, NW_018654712.1, NW_018654713.1, NW_018654714.1, NW_018654715.1, NW_018654716.1, NW_018654717.1, NW_018654718.1, NW_018654719.1, NW_018654720.1, NW_018654721.1, NW_018654722.1, NW_018654723.1, NW_018654724.1, NW_018654725.1, NW_018654726.1, NW_019805487.1, NW_019805488.1, NW_019805489.1, NW_019805490.1, NW_019805491.1, NW_019805492.1, NW_019805493.1, NW_019805494.1, NW_019805495.1, NW_019805496.1, NW_019805497.1, NW_019805498.1, NW_019805499.1, NW_019805500.1, NW_019805501.1, NW_019805502.1, NW_019805503.1, NW_021159987.1, NW_021159988.1, NW_021159989.1, NW_021159990.1, NW_021159991.1, NW_021159992.1, NW_021159993.1, NW_021159994.1, NW_021159995.1, NW_021159996.1, NW_021159997.1, NW_021159998.1, NW_021159999.1, NW_021160000.1, NW_021160001.1, NW_021160002.1, NW_021160003.1, NW_021160004.1, NW_021160005.1, NW_021160006.1, NW_021160007.1, NW_021160008.1, NW_021160009.1, NW_021160010.1, NW_021160011.1, NW_021160012.1, NW_021160014.1, NW_021160015.1, NW_021160016.1, NW_021160017.1, NW_021160019.1, NW_021160020.1, NW_021160021.1, NW_021160022.1, NW_021160023.1, NW_021160024.1, NW_021160025.1, NW_021160026.1, NW_021160027.1, NW_021160028.1, NW_021160029.1, NW_021160030.1, NW_021160031.1, NW_025791753.1, NW_025791754.1, NW_025791755.1, NW_025791756.1, NW_025791757.1, NW_025791758.1, NW_025791759.1, NW_025791760.1, NW_025791761.1, NW_025791762.1, NW_025791763.1, NW_025791764.1, NW_025791765.1, NW_025791766.1, NW_025791767.1, NW_025791768.1, NW_025791769.1, NW_025791770.1, NW_025791771.1, NW_025791772.1, NW_025791773.1, NW_025791774.1, NW_025791775.1, NW_025791776.1, NW_025791777.1, NW_025791778.1, NW_025791779.1, NW_025791780.1, NW_025791781.1, NW_025791782.1, NW_025791783.1, NW_025791784.1, NW_025791785.1, NW_025791786.1, NW_025791787.1, NW_025791788.1, NW_025791789.1, NW_025791790.1, NW_025791791.1, NW_025791792.1, NW_025791793.1, NW_025791794.1, NW_025791795.1, NW_025791796.1, NW_025791797.1, NW_025791798.1, NW_025791799.1, NW_025791800.1, NW_025791801.1, NW_025791802.1, NW_025791803.1, NW_025791804.1, NW_025791805.1, NW_025791806.1, NW_025791807.1, NW_025791808.1, NW_025791809.1, NW_025791810.1, NW_025791811.1, NW_025791812.1, NW_025791813.1, NW_025791814.1, NW_025791815.1, NW_025791816.1, NW_025791817.1, NW_025791818.1, NW_025791819.1, NW_025791820.1, NW_025791821.1]

***********************************************************************
Set the system property GATK_STACKTRACE_ON_USER_EXCEPTION (--java-options '-DGATK_STACKTRACE_ON_USER_EXCEPTION=true') to print the stack trace.
Error: Failed to create recalibration table for G2110067M_GALC. Skipping...
Adding Read Group information to G2110067_GALC...
INFO    2025-02-07 09:02:32     AddOrReplaceReadGroups

********** NOTE: Picard's command line syntax is changing.
**********
********** For more information, please see:
********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)
**********
********** The command line looks like this in the new syntax:
**********
**********    AddOrReplaceReadGroups -I /mnt/e/CSW/GALC/G2110067_GALC_region.bam -O /mnt/e/CSW/GALC/G2110067_GALC_rg.bam -RGID G2110067_GALC -RGLB lib1 -RGPL ILLUMINA -RGPU unit1 -RGSM G2110067_GALC
**********


09:02:32.972 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/home/rare1/gene/SNUM/picard-2.25.0/picard.jar!/com/intel/gkl/native/libgkl_compression.so
[Fri Feb 07 09:02:33 GMT 2025] AddOrReplaceReadGroups INPUT=/mnt/e/CSW/GALC/G2110067_GALC_region.bam OUTPUT=/mnt/e/CSW/GALC/G2110067_GALC_rg.bam RGID=G2110067_GALC RGLB=lib1 RGPL=ILLUMINA RGPU=unit1 RGSM=G2110067_GALC    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false
[Fri Feb 07 09:02:33 GMT 2025] Executing as rare1@DESKTOP-Q8E0MN3 on Linux 4.4.0-17134-Microsoft amd64; OpenJDK 64-Bit Server VM 22.0.1-internal-adhoc.conda.src; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.25.0
INFO    2025-02-07 09:02:33     AddOrReplaceReadGroups  Created read-group ID=G2110067_GALC PL=ILLUMINA LB=lib1 SM=G2110067_GALC

[Fri Feb 07 09:02:33 GMT 2025] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 0.01 minutes.
Runtime.totalMemory()=536870912
Running GATK MarkDuplicates for G2110067_GALC...
09:02:34.910 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/home/rare1/tool/gatk-4.2.0.0/gatk-package-4.2.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so
[Fri Feb 07 09:02:34 GMT 2025] MarkDuplicates --INPUT /mnt/e/CSW/GALC/G2110067_GALC_rg.bam --OUTPUT /mnt/e/CSW/GALC/G2110067_GALC_dedup.bam --METRICS_FILE /mnt/e/CSW/GALC/G2110067_GALC_markdup_metrics.txt --CREATE_INDEX true --MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP 50000 --MAX_FILE_HANDLES_FOR_READ_ENDS_MAP 8000 --SORTING_COLLECTION_SIZE_RATIO 0.25 --TAG_DUPLICATE_SET_MEMBERS false --REMOVE_SEQUENCING_DUPLICATES false --TAGGING_POLICY DontTag --CLEAR_DT true --DUPLEX_UMI false --ADD_PG_TAG_TO_READS true --REMOVE_DUPLICATES false --ASSUME_SORTED false --DUPLICATE_SCORING_STRATEGY SUM_OF_BASE_QUALITIES --PROGRAM_RECORD_ID MarkDuplicates --PROGRAM_GROUP_NAME MarkDuplicates --READ_NAME_REGEX <optimized capture of last three ':' separated fields as numeric values> --OPTICAL_DUPLICATE_PIXEL_DISTANCE 100 --MAX_OPTICAL_DUPLICATE_SET_SIZE 300000 --VERBOSITY INFO --QUIET false --VALIDATION_STRINGENCY STRICT --COMPRESSION_LEVEL 2 --MAX_RECORDS_IN_RAM 500000 --CREATE_MD5_FILE false --GA4GH_CLIENT_SECRETS client_secrets.json --help false --version false --showHidden false --USE_JDK_DEFLATER false --USE_JDK_INFLATER false
Feb 07, 2025 9:02:35 AM shaded.cloud_nio.com.google.auth.oauth2.ComputeEngineCredentials runningOnComputeEngine
INFO: Failed to detect whether we are running on Google Compute Engine.
[Fri Feb 07 09:02:35 GMT 2025] Executing as rare1@DESKTOP-Q8E0MN3 on Linux 4.4.0-17134-Microsoft amd64; OpenJDK 64-Bit Server VM 22.0.1-internal-adhoc.conda.src; Deflater: Intel; Inflater: Intel; Provider GCS is available; Picard version: Version:4.2.0.0
INFO    2025-02-07 09:02:35     MarkDuplicates  Start of doWork freeMemory: 83838696; totalMemory: 113246208; maxMemory: 8522825728
INFO    2025-02-07 09:02:35     MarkDuplicates  Reading input file and constructing read end information.
INFO    2025-02-07 09:02:35     MarkDuplicates  Will retain up to 30879803 data points before spilling to disk.
INFO    2025-02-07 09:02:35     MarkDuplicates  Read 14411 records. 240 pairs never matched.
INFO    2025-02-07 09:02:35     MarkDuplicates  After buildSortedReadEndLists freeMemory: 227882568; totalMemory: 511705088; maxMemory: 8522825728
INFO    2025-02-07 09:02:35     MarkDuplicates  Will retain up to 266338304 duplicate indices before spilling to disk.
INFO    2025-02-07 09:02:36     MarkDuplicates  Traversing read pair information and detecting duplicates.
INFO    2025-02-07 09:02:36     MarkDuplicates  Traversing fragment information and detecting duplicates.
INFO    2025-02-07 09:02:36     MarkDuplicates  Sorting list of duplicate records.
INFO    2025-02-07 09:02:36     MarkDuplicates  After generateDuplicateIndexes freeMemory: 1911329896; totalMemory: 4076863488; maxMemory: 8522825728
INFO    2025-02-07 09:02:36     MarkDuplicates  Marking 0 records as duplicates.
INFO    2025-02-07 09:02:36     MarkDuplicates  Found 0 optical duplicate clusters.
INFO    2025-02-07 09:02:36     MarkDuplicates  Reads are assumed to be ordered by: coordinate
INFO    2025-02-07 09:02:37     MarkDuplicates  Writing complete. Closing input iterator.
INFO    2025-02-07 09:02:37     MarkDuplicates  Duplicate Index cleanup.
INFO    2025-02-07 09:02:37     MarkDuplicates  Getting Memory Stats.
INFO    2025-02-07 09:02:37     MarkDuplicates  Before output close freeMemory: 272226376; totalMemory: 310378496; maxMemory: 8522825728
INFO    2025-02-07 09:02:37     MarkDuplicates  Closed outputs. Getting more Memory Stats.
INFO    2025-02-07 09:02:37     MarkDuplicates  After output close freeMemory: 260529000; totalMemory: 293601280; maxMemory: 8522825728
[Fri Feb 07 09:02:37 GMT 2025] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 0.04 minutes.
Runtime.totalMemory()=293601280
Tool returned:
0
Running GATK BaseRecalibrator for G2110067_GALC...
09:02:39.049 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/home/rare1/tool/gatk-4.2.0.0/gatk-package-4.2.0.0-local.jar!/com/intel/gkl/native/libgkl_compression.so
Feb 07, 2025 9:02:39 AM shaded.cloud_nio.com.google.auth.oauth2.ComputeEngineCredentials runningOnComputeEngine
INFO: Failed to detect whether we are running on Google Compute Engine.
09:02:39.194 INFO  BaseRecalibrator - ------------------------------------------------------------
09:02:39.194 INFO  BaseRecalibrator - The Genome Analysis Toolkit (GATK) v4.2.0.0
09:02:39.195 INFO  BaseRecalibrator - For support and documentation go to https://software.broadinstitute.org/gatk/
09:02:39.195 INFO  BaseRecalibrator - Executing as rare1@DESKTOP-Q8E0MN3 on Linux v4.4.0-17134-Microsoft amd64
09:02:39.196 INFO  BaseRecalibrator - Java runtime: OpenJDK 64-Bit Server VM v22.0.1-internal-adhoc.conda.src
09:02:39.197 INFO  BaseRecalibrator - Start Date/Time: February 7, 2025, 9:02:39‚ÄØAM GMT
09:02:39.197 INFO  BaseRecalibrator - ------------------------------------------------------------
09:02:39.198 INFO  BaseRecalibrator - ------------------------------------------------------------
09:02:39.199 INFO  BaseRecalibrator - HTSJDK Version: 2.24.0
09:02:39.199 INFO  BaseRecalibrator - Picard Version: 2.25.0
09:02:39.200 INFO  BaseRecalibrator - Built for Spark Version: 2.4.5
09:02:39.203 INFO  BaseRecalibrator - HTSJDK Defaults.COMPRESSION_LEVEL : 2
09:02:39.204 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_READ_FOR_SAMTOOLS : false
09:02:39.204 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_SAMTOOLS : true
09:02:39.205 INFO  BaseRecalibrator - HTSJDK Defaults.USE_ASYNC_IO_WRITE_FOR_TRIBBLE : false
09:02:39.206 INFO  BaseRecalibrator - Deflater: IntelDeflater
09:02:39.206 INFO  BaseRecalibrator - Inflater: IntelInflater
09:02:39.207 INFO  BaseRecalibrator - GCS max retries/reopens: 20
09:02:39.207 INFO  BaseRecalibrator - Requester pays: disabled
09:02:39.208 INFO  BaseRecalibrator - Initializing engine
09:02:39.360 INFO  FeatureManager - Using codec VCFCodec to read file file:///mnt/e/CSW/reference/GCF_000001405.40.gz
09:02:39.435 WARN  IndexUtils - Feature file "file:///mnt/e/CSW/reference/GCF_000001405.40.gz" appears to contain no sequence dictionary. Attempting to retrieve a sequence dictionary from the associated index file
09:02:39.557 INFO  BaseRecalibrator - Shutting down engine
[February 7, 2025, 9:02:39‚ÄØAM GMT] org.broadinstitute.hellbender.tools.walkers.bqsr.BaseRecalibrator done. Elapsed time: 0.01 minutes.
Runtime.totalMemory()=8589934592
***********************************************************************

A USER ERROR has occurred: Input files reference and features have incompatible contigs: No overlapping contigs found.
  reference contigs = [chr14]
  features contigs = [NC_000001.11, NC_000002.12, NC_000003.12, NC_000004.12, NC_000005.10, NC_000006.12, NC_000007.14, NC_000008.11, NC_000009.12, NC_000010.11, NC_000011.10, NC_000012.12, NC_000013.11, NC_000014.9, NC_000015.10, NC_000016.10, NC_000017.11, NC_000018.10, NC_000019.10, NC_000020.11, NC_000021.9, NC_000022.11, NC_000023.11, NC_000024.10, NC_012920.1, NT_113793.3, NT_113796.3, NT_113888.1, NT_113889.1, NT_113891.3, NT_113901.1, NT_113930.2, NT_113948.1, NT_113949.2, NT_167208.1, NT_167209.1, NT_167211.2, NT_167213.1, NT_167214.1, NT_167215.1, NT_167218.1, NT_167219.1, NT_167220.1, NT_167244.2, NT_167245.2, NT_167246.2, NT_167247.2, NT_167248.2, NT_167249.2, NT_167250.2, NT_167251.2, NT_187361.1, NT_187362.1, NT_187363.1, NT_187364.1, NT_187365.1, NT_187366.1, NT_187367.1, NT_187368.1, NT_187369.1, NT_187370.1, NT_187371.1, NT_187372.1, NT_187373.1, NT_187374.1, NT_187375.1, NT_187377.1, NT_187378.1, NT_187379.1, NT_187380.1, NT_187381.1, NT_187382.1, NT_187383.1, NT_187384.1, NT_187385.1, NT_187386.1, NT_187387.1, NT_187388.1, NT_187390.1, NT_187391.1, NT_187392.1, NT_187393.1, NT_187394.1, NT_187395.1, NT_187396.1, NT_187397.1, NT_187398.1, NT_187399.1, NT_187400.1, NT_187401.1, NT_187402.1, NT_187403.1, NT_187404.1, NT_187405.1, NT_187406.1, NT_187407.1, NT_187408.1, NT_187409.1, NT_187410.1, NT_187411.1, NT_187412.1, NT_187413.1, NT_187414.1, NT_187415.1, NT_187416.1, NT_187417.1, NT_187418.1, NT_187419.1, NT_187420.1, NT_187421.1, NT_187422.1, NT_187423.1, NT_187424.1, NT_187425.1, NT_187426.1, NT_187427.1, NT_187428.1, NT_187429.1, NT_187430.1, NT_187431.1, NT_187432.1, NT_187433.1, NT_187434.1, NT_187435.1, NT_187436.1, NT_187437.1, NT_187438.1, NT_187439.1, NT_187440.1, NT_187441.1, NT_187442.1, NT_187443.1, NT_187444.1, NT_187446.1, NT_187447.1, NT_187448.1, NT_187449.1, NT_187450.1, NT_187451.1, NT_187452.1, NT_187453.1, NT_187454.1, NT_187455.1, NT_187456.1, NT_187457.1, NT_187458.1, NT_187459.1, NT_187460.1, NT_187461.1, NT_187462.1, NT_187464.1, NT_187465.1, NT_187466.1, NT_187467.1, NT_187468.1, NT_187469.1, NT_187470.1, NT_187471.1, NT_187473.1, NT_187474.1, NT_187475.1, NT_187476.1, NT_187477.1, NT_187478.1, NT_187479.1, NT_187480.1, NT_187481.1, NT_187482.1, NT_187483.1, NT_187484.1, NT_187485.1, NT_187486.1, NT_187487.1, NT_187488.1, NT_187490.1, NT_187491.1, NT_187493.1, NT_187494.1, NT_187495.1, NT_187496.1, NT_187497.1, NT_187498.1, NT_187499.1, NT_187500.1, NT_187501.1, NT_187502.1, NT_187503.1, NT_187504.1, NT_187505.1, NT_187506.1, NT_187508.1, NT_187509.1, NT_187510.1, NT_187511.1, NT_187512.1, NT_187513.1, NT_187514.1, NT_187515.1, NT_187516.1, NT_187517.1, NT_187518.1, NT_187519.1, NT_187520.1, NT_187521.1, NT_187522.1, NT_187523.1, NT_187524.1, NT_187525.1, NT_187526.1, NT_187527.1, NT_187528.1, NT_187529.1, NT_187530.1, NT_187531.1, NT_187532.1, NT_187533.1, NT_187534.1, NT_187535.1, NT_187536.1, NT_187537.1, NT_187538.1, NT_187539.1, NT_187540.1, NT_187541.1, NT_187542.1, NT_187543.1, NT_187544.1, NT_187545.1, NT_187546.1, NT_187547.1, NT_187548.1, NT_187549.1, NT_187550.1, NT_187551.1, NT_187552.1, NT_187553.1, NT_187554.1, NT_187555.1, NT_187556.1, NT_187557.1, NT_187558.1, NT_187559.1, NT_187560.1, NT_187561.1, NT_187562.1, NT_187563.1, NT_187564.1, NT_187565.1, NT_187566.1, NT_187567.1, NT_187568.1, NT_187569.1, NT_187570.1, NT_187571.1, NT_187572.1, NT_187573.1, NT_187574.1, NT_187575.1, NT_187576.1, NT_187577.1, NT_187578.1, NT_187579.1, NT_187581.1, NT_187582.1, NT_187583.1, NT_187584.1, NT_187585.1, NT_187586.1, NT_187587.1, NT_187588.1, NT_187589.1, NT_187590.1, NT_187591.1, NT_187592.1, NT_187593.1, NT_187594.1, NT_187595.1, NT_187596.1, NT_187597.1, NT_187598.1, NT_187599.1, NT_187600.1, NT_187601.1, NT_187602.1, NT_187603.1, NT_187604.1, NT_187605.1, NT_187606.1, NT_187607.1, NT_187608.1, NT_187609.1, NT_187610.1, NT_187611.1, NT_187612.1, NT_187613.1, NT_187614.1, NT_187615.1, NT_187616.1, NT_187617.1, NT_187618.1, NT_187619.1, NT_187620.1, NT_187621.1, NT_187622.1, NT_187623.1, NT_187624.1, NT_187625.1, NT_187626.1, NT_187627.1, NT_187628.1, NT_187629.1, NT_187630.1, NT_187631.1, NT_187632.1, NT_187633.1, NT_187634.1, NT_187635.1, NT_187636.1, NT_187637.1, NT_187638.1, NT_187639.1, NT_187640.1, NT_187641.1, NT_187642.1, NT_187643.1, NT_187644.1, NT_187645.1, NT_187646.1, NT_187647.1, NT_187648.1, NT_187649.1, NT_187650.1, NT_187651.1, NT_187652.1, NT_187653.1, NT_187654.1, NT_187655.1, NT_187656.1, NT_187657.1, NT_187658.1, NT_187659.1, NT_187660.1, NT_187661.1, NT_187662.1, NT_187663.1, NT_187664.1, NT_187665.1, NT_187666.1, NT_187667.1, NT_187668.1, NT_187669.1, NT_187670.1, NT_187671.1, NT_187672.1, NT_187673.1, NT_187674.1, NT_187675.1, NT_187676.1, NT_187677.1, NT_187678.1, NT_187679.1, NT_187680.1, NT_187681.1, NT_187682.1, NT_187683.1, NT_187684.1, NT_187685.1, NT_187686.1, NT_187687.1, NT_187688.1, NT_187689.1, NT_187690.1, NT_187691.1, NT_187692.1, NT_187693.1, NW_003315905.1, NW_003315906.1, NW_003315907.2, NW_003315908.1, NW_003315909.1, NW_003315913.1, NW_003315914.1, NW_003315915.1, NW_003315917.2, NW_003315918.1, NW_003315919.1, NW_003315920.1, NW_003315921.1, NW_003315922.2, NW_003315928.1, NW_003315929.1, NW_003315930.1, NW_003315931.1, NW_003315934.1, NW_003315935.1, NW_003315936.1, NW_003315938.1, NW_003315939.2, NW_003315940.1, NW_003315941.1, NW_003315942.2, NW_003315943.1, NW_003315944.2, NW_003315945.1, NW_003315946.1, NW_003315952.3, NW_003315953.2, NW_003315954.1, NW_003315955.1, NW_003315956.1, NW_003315957.1, NW_003315958.1, NW_003315959.1, NW_003315960.1, NW_003315961.1, NW_003315962.1, NW_003315963.1, NW_003315964.2, NW_003315965.1, NW_003315966.2, NW_003315967.2, NW_003315968.2, NW_003315969.2, NW_003315970.2, NW_003315971.2, NW_003315972.2, NW_003571033.2, NW_003571036.1, NW_003571049.1, NW_003571050.1, NW_003571054.1, NW_003571055.2, NW_003571056.2, NW_003571057.2, NW_003571058.2, NW_003571059.2, NW_003571060.1, NW_003571061.2, NW_003871060.2, NW_003871073.1, NW_003871074.1, NW_003871091.1, NW_003871092.1, NW_003871093.1, NW_004166862.2, NW_004504305.1, NW_009646194.1, NW_009646195.1, NW_009646196.1, NW_009646197.1, NW_009646198.1, NW_009646199.1, NW_009646200.1, NW_009646201.1, NW_009646202.1, NW_009646203.1, NW_009646204.1, NW_009646205.1, NW_009646206.1, NW_009646207.1, NW_009646208.1, NW_009646209.1, NW_011332687.1, NW_011332688.1, NW_011332689.1, NW_011332690.1, NW_011332691.1, NW_011332692.1, NW_011332693.1, NW_011332694.1, NW_011332695.1, NW_011332696.1, NW_011332697.1, NW_011332698.1, NW_011332699.1, NW_011332700.1, NW_011332701.1, NW_012132914.1, NW_012132915.1, NW_012132916.1, NW_012132917.1, NW_012132918.1, NW_012132919.1, NW_012132920.1, NW_012132921.1, NW_013171799.1, NW_013171800.1, NW_013171801.1, NW_013171802.1, NW_013171803.1, NW_013171804.1, NW_013171805.1, NW_013171806.1, NW_013171807.1, NW_013171808.1, NW_013171809.1, NW_013171810.1, NW_013171811.1, NW_013171812.1, NW_013171813.1, NW_013171814.1, NW_014040925.1, NW_014040926.1, NW_014040927.1, NW_014040928.1, NW_014040929.1, NW_014040930.1, NW_014040931.1, NW_015148966.2, NW_015148967.1, NW_015148968.1, NW_015148969.2, NW_015495298.1, NW_015495299.1, NW_015495300.1, NW_015495301.1, NW_016107297.1, NW_016107298.1, NW_016107299.1, NW_016107300.1, NW_016107301.1, NW_016107302.1, NW_016107303.1, NW_016107304.1, NW_016107305.1, NW_016107306.1, NW_016107307.1, NW_016107308.1, NW_016107309.1, NW_016107310.1, NW_016107311.1, NW_016107312.1, NW_016107313.1, NW_016107314.1, NW_017363813.1, NW_017363814.1, NW_017363815.1, NW_017363816.1, NW_017363817.1, NW_017363818.1, NW_017363819.1, NW_017363820.1, NW_017852928.1, NW_017852929.1, NW_017852930.1, NW_017852931.1, NW_017852932.1, NW_017852933.1, NW_018654706.1, NW_018654707.1, NW_018654708.1, NW_018654709.1, NW_018654710.1, NW_018654711.1, NW_018654712.1, NW_018654713.1, NW_018654714.1, NW_018654715.1, NW_018654716.1, NW_018654717.1, NW_018654718.1, NW_018654719.1, NW_018654720.1, NW_018654721.1, NW_018654722.1, NW_018654723.1, NW_018654724.1, NW_018654725.1, NW_018654726.1, NW_019805487.1, NW_019805488.1, NW_019805489.1, NW_019805490.1, NW_019805491.1, NW_019805492.1, NW_019805493.1, NW_019805494.1, NW_019805495.1, NW_019805496.1, NW_019805497.1, NW_019805498.1, NW_019805499.1, NW_019805500.1, NW_019805501.1, NW_019805502.1, NW_019805503.1, NW_021159987.1, NW_021159988.1, NW_021159989.1, NW_021159990.1, NW_021159991.1, NW_021159992.1, NW_021159993.1, NW_021159994.1, NW_021159995.1, NW_021159996.1, NW_021159997.1, NW_021159998.1, NW_021159999.1, NW_021160000.1, NW_021160001.1, NW_021160002.1, NW_021160003.1, NW_021160004.1, NW_021160005.1, NW_021160006.1, NW_021160007.1, NW_021160008.1, NW_021160009.1, NW_021160010.1, NW_021160011.1, NW_021160012.1, NW_021160014.1, NW_021160015.1, NW_021160016.1, NW_021160017.1, NW_021160019.1, NW_021160020.1, NW_021160021.1, NW_021160022.1, NW_021160023.1, NW_021160024.1, NW_021160025.1, NW_021160026.1, NW_021160027.1, NW_021160028.1, NW_021160029.1, NW_021160030.1, NW_021160031.1, NW_025791753.1, NW_025791754.1, NW_025791755.1, NW_025791756.1, NW_025791757.1, NW_025791758.1, NW_025791759.1, NW_025791760.1, NW_025791761.1, NW_025791762.1, NW_025791763.1, NW_025791764.1, NW_025791765.1, NW_025791766.1, NW_025791767.1, NW_025791768.1, NW_025791769.1, NW_025791770.1, NW_025791771.1, NW_025791772.1, NW_025791773.1, NW_025791774.1, NW_025791775.1, NW_025791776.1, NW_025791777.1, NW_025791778.1, NW_025791779.1, NW_025791780.1, NW_025791781.1, NW_025791782.1, NW_025791783.1, NW_025791784.1, NW_025791785.1, NW_025791786.1, NW_025791787.1, NW_025791788.1, NW_025791789.1, NW_025791790.1, NW_025791791.1, NW_025791792.1, NW_025791793.1, NW_025791794.1, NW_025791795.1, NW_025791796.1, NW_025791797.1, NW_025791798.1, NW_025791799.1, NW_025791800.1, NW_025791801.1, NW_025791802.1, NW_025791803.1, NW_025791804.1, NW_025791805.1, NW_025791806.1, NW_025791807.1, NW_025791808.1, NW_025791809.1, NW_025791810.1, NW_025791811.1, NW_025791812.1, NW_025791813.1, NW_025791814.1, NW_025791815.1, NW_025791816.1, NW_025791817.1, NW_025791818.1, NW_025791819.1, NW_025791820.1, NW_025791821.1]

***********************************************************************
Set the system property GATK_STACKTRACE_ON_USER_EXCEPTION (--java-options '-DGATK_STACKTRACE_ON_USER_EXCEPTION=true') to print the stack trace.
Error: Failed to create recalibration table for G2110067_GALC. Skipping...
All GATK analyses are complete.
(base) rare1@DESKTOP-Q8E0MN3:/mnt/e/CSW/GALC/GATK$ Ïù¥Î†áÍ≤åÎñ† ÏôúÍ∑∏Îü¨ÎäîÍ±∞Ïïº?
